### PR TITLE
Completed styling of the website

### PR DESCRIPTION
**Pull request by AsimKhan2019**
I have applied the styling to the website, although not close to the zeplin design near to its concept. The design implementation was challenging.

### CODE REVIEW

File: src/components/Book.js [hunk 1]
```diff
@@ -1,18 +1,42 @@
...
+        <div className="chapters">
+          <span className="left title">Current Chapter</span>
+          <div className="clr" />
+          <span className="left">
+            Chapter
+            {Math.floor(Math.random() * 10)}
+          </span>
+          <div className="clr" />
+          <span className="left"><button className="chapterbutton" type="button">Update Chapter</button></span>
+        </div>
```
Comment (ahmetbozaci) [inline src/components/Book.js]: - [x] As you can see in the original design, your design little different from that. According to [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/styling.md#:~:text=Your%20final%20design%20should%20match%20Zeplin%27s%20design.), your design should match to the original design. Kindly change it.
image

### ORIGINAL
image

File: src/index.css [hunk 2]
```diff
@@ -74,3 +74,137 @@ body {
...
+}
+
+.remove {
+  background: linear-gradient(to bottom right, #ef4765, #ff9a5a);
+  border: 0;
+  border-radius: 12px;
+  color: #fff;
+  cursor: pointer;
+  display: inline-block;
+  font-family: -apple-system, system-ui, "Segoe UI", Roboto, Helvetica, Arial, sans-serif;
```
Comment (ahmetbozaci) [inline src/index.css]: - [x] The Font family is not this in the original design. Kindly check the original [design](https://app.zeplin.io/project/5b35a9e13227086040f8eb75/screen/5b695e29bb8c844f118f9378) and update your design.

**Review by ahmetbozaci — CHANGES_REQUESTED** [review]
# ♻️♻️♻️ STATUS: Changes Required  ♻️♻️♻️


 @AsimKhan2019  ✋🏼 

#### Your project is great. You are almost done. However, there are some issues you should fix. Please fix them then your project will be approved.

___ 

### 💯  Highlights 💯
- Your git-flow is correct. ✔️
- You set up linter files and no errors. ✔️
- Your pull request is very professional. It has a meaningful title and summary. Keep it up.✔️
___


### ♻️ Request changes ♻️

- [x] _Check the comments under the review._

___

### Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
⚠️ @AsimKhan2019 ⚠️ 
- [x] Adding and removing books is not working well. I have to refresh the page to see what changes. I recommend fixing this.
___

### Cheers and Happy coding! 💻 👏
almost


___

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification._

_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._


___

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form.](https://airtable.com/shrQAqnBwek5a0O0s)_

[GitHub](https://github.com/ahmetbozaci) | [LinkedIn](https://linkedin.com/in/ahmetbozaci) | [Twitter](https://twitter.com/ahmtbozaci)

_Reviewed by [Ahmet Bozacı ](https://github.com/ahmetbozaci)_

File: src/index.css (hunk 2 above)
Comment (AsimKhan2019) [reply src/index.css]: @ahmetbozaci Done

File: src/components/Book.js (hunk 1 above)
Comment (AsimKhan2019) [reply src/components/Book.js]: @ahmetbozaci Done

**Review by rustam-tolipov — APPROVED** [review]
## Hello @AsimKhan2019 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 🎉 🎉 🎉 🎉 

Cheers and Happy coding!👏👏👏

Developer-512px-10

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Comment by rustam-tolipov** [comment]
- [ ] [OPTIONAL] Please don't forget to add the MIT license file  to your project 📄

**Comment by AsimKhan2019** [comment]
@Rustamxon7 Thank you for the approval

**Pull request merged**